File: ServerApp/src/Business/Department.java
package Business;

public class Department {

    String nameOfDepartment;
    String fioManager;
    String managerStartDate;
    String streetAddress;
    int postalCode;
    String city;
    String region;
    int countEmployee;

    public Department(String nameOfDepartment, String fioManager, String managerStartDate, String streetAddress, int postalCode, String city, String region, int countEmployee) {
        this.nameOfDepartment = nameOfDepartment;
        this.fioManager = fioManager;
        this.managerStartDate = managerStartDate;
        this.streetAddress = streetAddress;
        this.postalCode = postalCode;
        this.city = city;
        this.region = region;
        this.countEmployee = countEmployee;
    }
}

File: AndroidApp/app/src/main/java/ru/biluta/androidapp/ui/business/Department.kt
package ru.biluta.androidapp.ui.business

data class Department(
    val nameOfDepartment: String,
    val fioManager: String,
    val managerStartDate: String,
    val streetAddress: String,
    val postalCode: Int,
    val city: String,
    val region: String,
    val countEmployee: Int
)
